prod/westeurope/internal/api/functions_assets_r3/function_app_slot_staging/terragrunt.hcl

```diff
@@ -88,15 +88,14 @@ inputs = {
     FETCH_KEEPALIVE_TIMEOUT             = "60000"
 
     SLOT_TASK_HUBNAME = "ProductionTaskHub"
```

Comment:
unrelated to this PR but this should be `StagingTaskHub`

Reply:
Done I changed that too